cli/actions/jwt_secrets.go

```diff
@@ -0,0 +1,58 @@
+package actions
+
+import (
+	"fmt"
+	"github.com/NethermindEth/sedge/configs"
+	"github.com/NethermindEth/sedge/internal/crypto"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"path/filepath"
+)
+
+type CreateJwtSecretOptions struct {
+	JwtPath        string
+	Network        string
+	GenerationPath string
+}
+
+func (s *sedgeActions) CreateJwtSecrets(options CreateJwtSecretOptions) (string, error) {
+	if options.Network == "" {
+		return "", fmt.Errorf("network not found")
```

Comment:
Put error message in `actions/errors.go` (yeah, for the Actions is different, @adriantpaez introduced this and we are now respecting it)